.gitignore

```diff
@@ -1,3 +1,4 @@
 composer.phar
 /vendor/
 composer.lock
+.idea
```

Comment:
I usually add these IDE-like files to my global `.gitignore` file, so I keep all editor/IDE related stuff out of the repo. Otherwise you end up adding this for every IDE people use. That is exhausting.